WWW/1_first_task.wdl

```diff
@@ -68,18 +53,18 @@ workflow minidata_test_alignment {
 task BwaMem {
   input {
     File input_fastq
-    String base_file_name
     File ref_fasta
     File ref_fasta_index
     File ref_dict
-    File? ref_alt
     File ref_amb
     File ref_ann
     File ref_bwt
     File ref_pac
     File ref_sa
-    String taskDocker
   }
+  
+  String base_file_name = basename(input_fastq, ".fastq")
```

Comment:
add a comment in the document to pay attention to the file extension, also note it can be made work with both with something like 

sub(sub(basename(input_fastq), ".fastq", ""), ".fq", "")